Footer mobile mode

helloimela opened the pull request:
**Describe pull-request**</br>
C-corporate-footer components dropup menu in mobile view is removed and improved style is implemented


**Solving issue**</br>
Fixes: #101

helloimela committed:
Remove dropup menu

- Improve styling in mobile mode
- Fixes #101

Commit:
Merge branch 'master' into improvement/footer_mobile_mode

Commit:
Change footer structure

- Remove uneccessary html structure
- Change styling for new structure

Commit:
Fix padding on footer without links or social media

- If footer does not contain links, do not render navbar for links

mmexvr reviewed (dismissed):
Seems in IE the border-radius isn't correct 

Master and local branch
![image](https://user-images.githubusercontent.com/35451568/57929498-ccaed600-78b3-11e9-985d-36566126dee3.png)

Commit:
Fix styling for IE

- Rounded social media was overruled by ::slotted (a), added a rule for social media 'a' element

mmexvr reviewed (changes requested):
- Scania-wordmark-symbol is cut off in IE 
- Hovering over social-media elements in slots seems to cause the height to change

Master has the same problems

Commit:
Fix footer styling

- Fix size of wordmark
- Fix white space div on link hover on IE

mmexvr reviewed (changes requested):
Still seems to be a cut in the scania logo in IE, have not looked at FF

![image](https://user-images.githubusercontent.com/35451568/58458108-9893aa80-8128-11e9-88cf-bec3b1938eef.png)
